sdk/nodejs/server.ts

```diff
@@ -1,3 +1,4 @@
+// TODO(TomChv): Is this file useful?
```

Comment:
it's used for extensions, which aren't released yet and will be majorly refactored for nodejs still. I'd say lets leave it around unless it's causing any problems in which case it's probably fine to remove for now.

Reply:
agreed. We'll take care of the clean up after the launch. I'll create issues.